gomock/controller.go

```diff
@@ -92,6 +113,8 @@ type Controller struct {
 	finished      bool
 }
 
+// NewController a new Controller. It is the preferred way to create a
```

Comment:
Is this missing a word? NewController creates a new Controller.

Reply:
Ah, thanks. I think it should be:

> NewController returns a new Controller.